ch-covidcertificate-backend-verifier/ch-covidcertificate-backend-verifier-valuesets/src/test/java/ch/admin/bag/covidcertificate/backend/verifier/valuesets/VerificationRulesTest.java

```diff
@@ -68,6 +74,18 @@ public void generateRulesJsons() throws Exception {
     private JsonNode mapMasterToV2() throws Exception {
         JsonNode master =
                 mapper.readTree(new ClassPathResource(RULES_MASTER_CLASSPATH).getInputStream());
+
+        List<JsonNode> rules = Arrays.stream(new ClassPathResource(RULES_MASTER_DIR).getFile().list())
```

Comment:
i'm not sure where the ordering of the rules gets scrambled. but i suspect it's in .getFile().list(). if possible it would be nice if we would conserve alphabetical ordering of rules by id. that way we have a deterministic way of producing the outputs, making the diffs easier to see